fast_versioning.gemspec

```diff
@@ -3,18 +3,18 @@ $:.push File.expand_path("../lib", __FILE__)
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
   s.name        = "fast_versioning"
-  s.version     = '0.1.2'
+  s.version     = '0.2.0'
   s.authors     = ['Arcadia Power', 'Iwo Dziechciarow', 'Justin Doody']
   s.email       = ['engineering@arcadiapower.com']
   s.homepage    = 'https://github.com/ArcadiaPower/fast-versioning'
   s.summary     = 'Fast versioning extension for paper_trail'
   s.description = 'Fast versioning extension for paper_trail'
   s.license     = 'MIT'
 
-  s.files = Dir['{app,config,db,lib}/**/*', 'Rakefile', 'README.md']
+  s.files = Dir['{app,db,lib}/**/*', 'Rakefile', 'README.md']
 
-  s.add_dependency 'rails'
-  s.add_dependency 'paper_trail', '~>4.2'
+  s.add_dependency 'rails', '~> 5.0.7'
+  s.add_dependency 'paper_trail', '~> 5.2'
```

Comment:
nothing you did here seems to make this rails 4, or paper_trail 4 incompatible, so maybe we should do:
```
```

Reply:
ah yeah fair enough -- it's all just dependent on any PaperTrail's api changes not rails -- was being extra paranoid but will switch back considering this is public facing

Reply:
hmm actually it looks like specs do fail on 5.2

Reply:
as this is just a temporary step (will be updating to higher papertrail versions soon i'm going to lock it to < 5.1